cardano-cli/src/Cardano/CLI/Types/Errors/RegistrationError.hs

```diff
@@ -27,3 +35,11 @@ instance Error RegistrationError where
       "Cannot read stake credential: " <> prettyError e
     RegistrationStakeError e ->
       "Stake address registation error: " <> prettyError e
+    RegistrationMismatchedDRepMetadataHashError expectedHash actualHash ->
+      "DRep metadata Hashes do not match!"
+        <> "\nExpected:"
+          <+> pretty (show (L.extractHash expectedHash))
+        <> "\n  Actual:"
```

Comment:
```suggestion
        <> "\nActual:"
```

Bit weird to indent the `Actual` case, whereas you don't indent the `Expected` case?

Reply:
See https://github.com/IntersectMBO/cardano-cli/pull/916#discussion_r1781299176